simplq/src/components/pages/Home/StaticInfos.jsx

```diff
@@ -24,26 +24,29 @@ export const ButtonGroup = (props) => (
 export const BenefitsInfo = () => (
   <div className={styles['benefits-info']}>
     <div>
-      {/* <img></img> */}
+      <img src="/images/minimize_crowding.svg" alt="mimimze crowding" />
       <p>Minimize crowding by keeping your queue virtual</p>
     </div>
     <div>
-      {/* <img></img> */}
+      <img src="/images/enforce_social_dist.svg" alt="Enforce social distancing" />
       <p>Enforce social distancing effectively</p>
     </div>
     <div>
-      {/* <img></img> */}
+      <img src="/images/check_status.svg " alt="Check your current status" />
       <p>Check your current status in the queue</p>
     </div>
   </div>
 );
 
 export const HowToCreate = () => (
   <div className={styles['create-queue']}>
-    <h2>Create a queue at the click of a button</h2>
+    <h2>Create a Queue at the click of a button</h2>
     <div>
-      <h3>Create Queue</h3>
-      <p>Enter queue name and create a queue totally free of cost</p>
+      <img src="/images/queue_creation.svg" alt="Create Queue" />
+      <div>
+        <h3>Create Queue</h3>
+        <p>Enter queue name and create a queue totally free of cost</p>
```

Comment:
Why are we capitalizing queue in line 43 but not here?

Reply:
🤦

Reply:
Done